cmd/createDeployPipelineJob.go

```diff
@@ -89,12 +97,16 @@ Examples:
 
 		triggerPipelineParams := application.NewTriggerPipelineDeployParams()
 		triggerPipelineParams.SetAppName(*appName)
-		triggerPipelineParams.SetPipelineParametersDeploy(&models.PipelineParametersDeploy{
+		parametersDeploy := models.PipelineParametersDeploy{
 			ToEnvironment: targetEnvironment,
 			ImageTagNames: imageTagNames,
 			TriggeredBy:   triggeredByUser,
 			CommitID:      commitID,
-		})
+		}
+		if components := strings.Join(componentsToDeploy, ","); len(components) > 0 {
```

Comment:
PipelineParametersDeploy.Components in radix-api should be a string slice, not a string. Then we won't need to generate a comma separated list oif the values when sending the request, and unwrap the comma separated string in radix-api

Reply:
Fixed